book_review_system/src/main/java/app/controller/RoleController.java

```diff
@@ -0,0 +1,29 @@
+package app.controller;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import app.model.Role;
+import app.service.RoleService;
+
+@Controller
+public class RoleController {
+	private static final Logger logger = Logger.getLogger(RoleController.class);
+	
+	@Autowired
+	private RoleService roleService;
+	
+	@RequestMapping("/role")
+	public ModelAndView loadRoles(){
+		logger.info("load Roles");
+		ModelAndView model = new ModelAndView("role");
+		model.addObject("roles", roleService.loadRoles());
+		for (Role role : roleService.loadRoles()) {
+			System.out.println(role.getName());
+		}
+		return model;
```

Comment:
xóa line 24 -> 26
mà `load_roles` để làm gì vậy em, có page view toàn bộ roles à, chị nghĩ view user thôi chứ ..ahihi

Reply:
Cái này dễ nhất em làm để fix hbm ấy chị. Có thời gian thì sau làm full chức năng này.

Reply:
ậc, chị ngẫm là ko có chức năng Role controller mô?
mà RolesController nhé - số nhiều á, sửa cái này nào